packages/cli/test/collect-puppeteer.test.js

```diff
@@ -78,7 +78,7 @@ describe('Lighthouse CI collect CLI with puppeteer', () => {
     const files = fs.readdirSync(path.join(autorunDir, '.lighthouseci'));
     const report = files.find(file => /lhr.*\.json$/.test(file));
     const lhr = JSON.parse(fs.readFileSync(path.join(autorunDir, '.lighthouseci', report)));
-    expect(lhr.userAgent).toContain('HeadlessChrome/111.0.5555.0'); // make sure the right chrome was used
+    expect(lhr.userAgent).toContain('Chrome/111.0.0.0'); // make sure the right chrome was used
```

Comment:
Needed because `--headless=new` no longer announces itself in the UA string.

From #336 the condition being asserted seems to be that puppeteer's Chrome was used, not that it was headless in particular (that's just the default without `headful: true` in the options), so this change seems ok.